src/utilities/view.ts

```diff
@@ -20,19 +19,17 @@ import {
  * Generates a default generic UI view.
  */
 export function generateDefaultView(
-  activityTypes: ActivityType[] = [],
   resourceTypes: ResourceType[] = [],
   externalEventTypes: ExternalEventType[] = [],
 ): View {
   const now = new Date().toISOString();
-  const types: string[] = activityTypes.map(({ name }) => name);
 
   const timeline = createTimeline([], { marginLeft: 250, marginRight: 30 });
   const timelines = [timeline];
 
   // Start with the activity row
   const activityLayer = createTimelineActivityLayer(timelines, {
-    filter: { activity: { static_types: types } },
+    filter: /* { activity: { static_types: types } } */ {},
```

Comment:
Small question (possibly moot), do we even need to pass an object to the `args` parameter?

Reply:
Oh good catch, shouldn't have to!

Reply:
Fixed